### PR TITLE
Remove machine leftovers

### PR DESCRIPTION
**Pull request by sed-i**
Cleanup machine leftovers.

In tandem with:
- https://github.com/canonical/grafana-agent-operator/pull/2

### CODE REVIEW

File: tests/scenario/test_start_statuses.py [hunk 1]
```diff
@@ -6,27 +6,24 @@
...
 def substrate(request):
     return request.param
 
 
 @pytest.fixture
 def charm_type(substrate) -> Type[CharmType]:
-    return {"lxd": machine_charm.GrafanaAgentMachineCharm, "k8s": k8s_charm.GrafanaAgentK8sCharm}[
-        substrate
-    ]
+    return {"k8s": charm.GrafanaAgentK8sCharm}[substrate]
```
Comment (dstathis) [inline tests/scenario/test_start_statuses.py:26]: Feels like this entire function should probably be removed. Or at the very least rewritten as `return charm.GrafanaAgentK8sCharm`.

File: tests/scenario/test_start_statuses.py [hunk 2]
```diff
@@ -53,13 +50,13 @@ def patch_all(substrate, placeholder_cfg_path):
...
     else:
-        with patch("k8s_charm.KubernetesServicePatch", lambda x, y: None):
+        with patch("charm.KubernetesServicePatch", lambda x, y: None):
             yield
 
 
 @pytest.fixture
 def charm_meta(substrate, charm_type) -> dict:
-    fname = {"lxd": "machine_metadata", "k8s": "k8s_metadata"}[substrate]
+    fname = {"k8s": "metadata"}[substrate]
```
Comment (dstathis) [inline tests/scenario/test_start_statuses.py:59]: Feels like this entire function should probably be removed. Or at the very least rewritten as `return "metadata"`.

**Review by dstathis — APPROVED** [review]

File: tests/scenario/test_start_statuses.py (hunk 1 above)
Comment (sed-i) [reply tests/scenario/test_start_statuses.py:26]: #256

File: tests/scenario/test_start_statuses.py (hunk 2 above)
Comment (sed-i) [reply tests/scenario/test_start_statuses.py:59]: #256

**Pull request merged**